en/setup/dev_env_linux_boutique.md

```diff
@@ -95,8 +95,10 @@ sudo yum install glibc.i686 ncurses-libs.i686
 
 <div class="host-code"></div>
 
+Ensure you have the multilib repository enabled.
+
 ```sh
-sudo pacman -S base-devel lib32-glibc git-core python-pyserial zip python-empy
+sudo pacman -S base-devel lib32-glibc git-core python-pyserial zip
```

Comment:
python-empy not found in my case. May be part of multilib, this issue was fixed before adding multilib repo